setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="vcf2fhir",
-    version="0.0.16",
+    version="0.0.17",

```

Comment:
Sorry missed this part, I didn't realize it before.
Can you help split the existing two commits in the same PR into 2 different PR's and create a new PR for setup.py changes? Totally we would have 3 different PR's out of this.

So next steps would be:
1. merge changelog for v0.0.1 to v0.0.16 (PR1)
2. merge version change in setup.py  (PR2)
2. Publish a new version to PyPi.  
3. Merge change log for v0.0.17 (PR3)

Thanks for all the help.

Reply:
Hi @srgothi92, I modified this PR to reflect the changes for `v0.0.1 - v0.0.16` only and created another [PR](https://github.com/elimuinformatics/vcf2fhir/pull/72) for the **version** change in `setup.py`. 

I think we would have to wait until this PR gets merged, before creating the PR for **CHANGELOG** updates corresponding to `v0.0.17`, otherwise it will have the same changes to `v0.0.1 - v0.0.16` part as this PR does.